_raw/locales/index.json

```diff
@@ -9,5 +9,5 @@
   { "code": "pt_BR", "name": "Português Brasileiro"},
   { "code": "ru", "name": "Русский"},
   { "code": "de", "name": "Deutsch" },
-  { "code": "jp", "name": "日本語" }
+  { "code": "ja", "name": "日本語" }
```

Comment:
Please don't change the language code, this will make users who selected jp meet some unexpected exceptions

Reply:
No, I won't. The language codes are defined by ISO 639.

https://www.loc.gov/standards/iso639-2/php/code_list.php

I made sure to fallback to the default language code `en`. No exception occurs. Users who selected Japanese can select it again.

Reply:
https://www.w3.org/WAI/ER/IG/ert/iso639.htm

https://www.iso.org/iso-639-language-code (purchase required)

Reply:
@vvvvvv1vvvvvv Could you understand? I feel like the identity as Japanese people has been denied.

Reply:
I can understand, probably we need another PR for language compatible, like user selected jp before but after update the code change to ja but jp is not exist
don't have enough time to think it currently
@dmy147 can you help to check it?

Reply:
Understood. I'll revert it and open another PR.